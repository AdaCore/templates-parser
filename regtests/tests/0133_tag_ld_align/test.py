from test_support import *

run('testme', ["tag_cr_align.tmplt"])
