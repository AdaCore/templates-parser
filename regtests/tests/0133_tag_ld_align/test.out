procedure Test is
begin
   A := A + 1;
   A := B + 5;
   A := A / 2;
end Test;

   procedure Test is
   begin
      A := A + 1;
      A := B + 5;
      A := A / 2;
   end Test;

procedure Test is
begin
   A := A + 1;
A := B + 5;
A := A / 2;
end Test;
